src/Http/Http.Abstractions/src/ProblemDetails/ProblemDetails.cs

```diff
@@ -64,8 +63,6 @@ public class ProblemDetails
     [JsonExtensionData]
     public IDictionary<string, object?> Extensions
     {
-        [RequiresUnreferencedCode("JSON serialization and deserialization of ProblemDetails.Extensions might require types that cannot be statically analyzed.")]
```

Comment:
Have you ensured that all `UnconditionalSuppressMessage` attributes that were added with this annotation as a justification are now removed?

I've seen most have been removed, but I haven't done a full search.